test/e2e-kuttl/quota-forest/09-install.yaml

```diff
@@ -0,0 +1,133 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+    - command: kubectl -n test delete appwrappers,jobs --all
```

Comment:
Nitpick, you should probably add a comment as to why are you deleting all the app wrappers that have previously been installed.